test/integrations/destinations/tiktok_ads/router/data.ts

```diff
@@ -2651,7 +2651,7 @@ export const data: RouterTestData[] = [
             {
```

Comment:
too many changes in test cases at the field level whereas changes are majorly on the batching logic - can you throw some light ?

Reply:
I have refactored the test case for minimal changes.